examples_tests/49.ComputeFFT/convolve.comp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#version 430 core
+
+#include <nbl/builtin/glsl/math/complex.glsl>
+
+layout(local_size_x=256, local_size_y=1, local_size_z=1) in;
+ 
+struct nbl_glsl_ext_FFT_output_t
+{
+	vec2 complex_value;
+};
```

Comment:
we'd usually do a "typedef" for this

in GLSL land it would be 
```glsl
#define nbl_glsl_ext_FFT_output_t vec2
```

Reply:
ah and one more thing... this code is not part of the engine, its part of your example... so just how in C++ you wouldn't start putting declarations from example source code into the `nbl::` namespaces, so should you not add `nbl_glsl_` prefixes to declarations of types and functions (unless you're "overriding", i.e. providing a definition for a function already declared in the builtin GLSL "engine space")